modules/sdk/src/main/scala/org/tessellation/sdk/http/routes/SnapshotRoutes.scala

```diff
@@ -91,6 +91,22 @@ final case class SnapshotRoutes[F[_]: Async: KryoSerializer, S <: Snapshot[_, _]
           }
         }.getOrElse(NotFound())
 
+    case GET -> Root / "hash" / SnapshotOrdinalVar(ordinal) :? FullSnapshotQueryParam(fullSnapshot) =>
+      import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+
+      if (!fullSnapshot) {
+        snapshotStorage
+          .getHash(ordinal)
+          .flatMap(Ok(_))
+      } else {
+        fullGlobalSnapshotStorage
+          .map(
+            _.readHash(ordinal)
+          )
+          .flatSequence
```

Comment:
if you use map then sequence, it can be a single operation like `traverse`, but if you use flatten after that, it can be a `flatTraverse`

Reply:
This has been changed to

```scala
        fullGlobalSnapshotStorage
          .flatTraverse(
            _.readHash(ordinal)
          )
          .flatMap(Ok(_))
```